src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/content.html.twig

```diff
@@ -0,0 +1,26 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{{ row[column.options.field]|raw }}
```

Comment:
I have the feeling that SimpleColumn (or DataColumn) is enough for that: don't you think?

Reply:
i was thinking the same, but instead of adding additional option to DataColumn i've implemented ContentColumn so it is explicit.

if you think its better to have it the same column, im okay with that. :)

Reply:
nah ok let's do that